### PR TITLE
Fix applications fetching at the Application List page

### PR DESCRIPTION
**Pull request by cakecatz**
**What this PR does / why we need it**:

Fix a bug that filter options were not applied when fetching applications after the application action. (EDIT, DELETE, DISABLE, etc.)

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/app/web/src/components/add-application-drawer/index.tsx [hunk 1]
```diff
@@ -22,14 +22,15 @@ import {
 export interface AddApplicationDrawerProps {
   open: boolean;
   onClose: () => void;
+  onAdded: () => void;
```
Comment (khanhtc1202) [inline pkg/app/web/src/components/add-application-drawer/index.tsx:25]: how about making this `onAdded` optional and make a default value for it on L33 👀 
In most of cases, looks like we only pass `onAdded={() => null}` for that 🤔

Comment (cakecatz) [reply pkg/app/web/src/components/add-application-drawer/index.tsx:25]: Sure, that might be a good idea.
Thank you!

**Comment by khanhtc1202** [comment]
Thank you 🚀 
/approve

**Pull request merged**